src/clj/game/cards/programs.clj

```diff
@@ -877,4 +877,48 @@
                    :counter-cost [:power 3]
                    :once :per-turn
                    :msg "gain [Click][Click]"
-                   :effect (effect (gain :click 2))}]}})
+                   :effect (effect (gain :click 2))}]}
+
+   "Wari"
+   {:events {:successful-run
+             {:interactive (req )
```

Comment:
This should be `(req true)` if you want this event to always force the Runner into using the simultaneous event resolution prompt.